src/org/labkey/test/components/ui/grids/ResponsiveGrid.java

```diff
@@ -522,6 +522,32 @@ public boolean gridMessagePresent()
                 .child(Locator.tagWithClass("div", "grid-message")).findElements(this));
     }
 
+    /** The responsiveGrid now supports redacting fields
+     *
+     * @param columnText the column name.  (uses starts-with matching)
+     * @return  true if the specified grid header cell has the 'phi-protected' class on it
+     */
+    public boolean getColumnPHIProtected(String columnText)
+    {
+        return elementCache().getColumnHeaderCell(columnText)
+                .getAttribute("class").contains("phi-protected");
+    }
+
+    /**
+     *  Gets the title attribute of the column header cell, if it has one
+     * @param columnText The text with which to find the cell (uses startswith matching)
+     * @return  the contents of the 'title' attribute of the cell, or null if the attribute is
+     * not present.
+     */
+    public String getColumnTitleAttribute(String columnText)
+    {
+        var cell = elementCache().getColumnHeaderCell(columnText);
+        if (null == cell.getAttribute("title"))
+            return null;
+        else
+            return cell.getAttribute("title");
```

Comment:
Why not just do:
`return elementCache().getColumnHeaderCell(columnText).getAttribute("title");`

Reply:
I think I was just worried about the attribute being null and trying to be clever
your suggestion is better, actually